packages/frontend/src/components/Strategies/Crab/CrabTrade.tsx

```diff
@@ -77,36 +77,55 @@ const CrabTrade: React.FC<CrabTradeType> = ({ maxCap, depositedAmount }) => {
     ethIndexPrice,
     calculateEthWillingToPay,
     calculateETHtoBorrowFromUniswap,
+    isTimeHedgeAvailable,
+    isPriceHedgeAvailable,
   } = useCrab()
   const { minCurrentUsd, minPnL, loading } = useCrabPosition(address || '')
   const { isRestricted } = useRestrictUser()
   const { ready } = useSqueethPool()
   const { dailyHistoricalFunding, currentImpliedFunding } = useController()
 
-  let maxCapError: string | undefined
-  let depositError: string | undefined
-  let withdrawError: string | undefined
-  let maxFlashCapError: string | undefined
-  let lowVolError: string | undefined
+  const { depositError, warning, withdrawError } = useMemo(() => {
+    let depositError: string | undefined
+    let withdrawError: string | undefined
+    let warning: string | undefined
 
-  if (connected) {
-    if (ethAmount.plus(depositedAmount).gte(maxCap)) {
-      maxCapError = 'Amount greater than strategy cap'
-    } else if (ethAmount.plus(depositedAmount).plus(borrowEth).gte(maxCap)) {
-      maxFlashCapError = `Amount greater than strategy cap since it flash borrows ${borrowEth.toFixed(
-        2,
-      )} ETH. Input a smaller amount`
+    if (connected) {
+      if (ethAmount.plus(depositedAmount).gte(maxCap)) {
+        depositError = 'Amount greater than strategy cap'
+      } else if (ethAmount.plus(depositedAmount).plus(borrowEth).gte(maxCap)) {
+        depositError = `Amount greater than strategy cap since it flash borrows ${borrowEth.toFixed(
+          2,
+        )} ETH. Input a smaller amount`
+      } else if (toTokenAmount(balance, 18).lt(ethAmount)) {
+        depositError = 'Insufficient ETH balance'
+      }
+      if (withdrawAmount.gt(currentEthValue)) {
+        withdrawError = 'Withdraw amount greater than strategy balance'
+      }
+      if (isTimeHedgeAvailable || isPriceHedgeAvailable) {
+        depositError = 'Deposits and withdraws available after the auction'
```

Comment:
```suggestion
        depositError = 'Deposits and withdraws available after the hedge auction'
```

Reply:
Just because users may not know there is an auction but more of them do know that the strategy hedges, so this explains why the auction